### DISCUSSION TITLE
https://www.xiaogenban1993.com/blog/24.07/java9%E6%A8%A1%E5%9D%97%E5%8C%96JPMS%E7%9A%84%E5%9D%91

### DISCUSSION THREAD

**Discussion by giscus** [sunwu51/notebook#64] [General]
# https://www.xiaogenban1993.com/blog/24.07/java9%E6%A8%A1%E5%9D%97%E5%8C%96JPMS%E7%9A%84%E5%9D%91

Generated by FreeCoder

https://www.xiaogenban1993.com/blog/24.07/java9%E6%A8%A1%E5%9D%97%E5%8C%96JPMS%E7%9A%84%E5%9D%91

**Comment by ftqiao** [comment]
你好。想了解下，一般开发人员都是“面向业务”。什么“工种”/场景会做这种相对来说比较“底层”的研究和学习？

**Reply by sunwu51** [reply]
从事基础平台建设的可能会有较多调研工作，比如内部jvm开发，k8s平台建设等。但更多的还是看自己兴趣。
